### DISCUSSION TITLE
Getting Error

### DISCUSSION THREAD

**Discussion by Jashandeep-1229** [netflie/whatsapp-cloud-api#13] [Q&A]
Client error: `POST https://graph.facebook.com/v13.0/101604565950496/messages` resulted in a `400 Bad Request` response: {"error":{"message":"An unknown error occurred","type":"OAuthException","code":1,"error_subcode":2593006,"is_transient": (truncated...
<img width="1197" alt="Screenshot 2022-07-01 at 12 48 57 AM" src="https://user-images.githubusercontent.com/67820986/176760071-c9bd893c-7e59-45c9-91c3-b60d58ea9097.png">
)

**Comment by aalbarca** [comment]
Please, follow the setup guide: https://developers.facebook.com/docs/whatsapp/cloud-api/get-started

**Reply by Jashandeep-1229** [reply]
How to register a phone number in WhatsApp business cloud api through laravel
